server/migrations/20170328005425-create-game.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
Do we need use strict on node JS? I thought that was only really needed for browsers